pg_search/Cargo.toml

```diff
@@ -39,8 +39,8 @@ serde = "1.0.188"
 serde_json = "1.0.105"
 serde_path_to_error = "0.1.14"
 shared = { path = "../shared" }
-tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy", rev = "c7f13ef" }
-tantivy-common = { git = "https://github.com/paradedb/tantivy.git", rev = "c7f13ef" }
+tantivy = { git = "https://github.com/paradedb/tantivy.git", package = "tantivy", rev = "4eab7b4" }
```

Comment:
heads up, I've cleaned up the history of `paradedb/tantivy` so you'll need to update this commit here. Sorry about that and thank you for doing this!